intellect-newgen-style.csl

```diff
@@ -0,0 +1,560 @@
+              <date-part name="year"/>
+            </date>
+          </if>
+          <else-if type="entry-dictionary" match="any">
+            <group delimiter=" ">
+              <date date-parts="year" form="text" variable="issued">
+                <date-part name="year"/>
+              </date>
+            </group>
+          </else-if>
+          <else-if type="manuscript" match="any">
+            <text term="forthcoming" suffix=" "/>
+            <date date-parts="year" form="text" variable="issued">
+              <date-part name="year"/>
+            </date>
+          </else-if>
+          <else>
+            <date date-parts="year" form="text" variable="issued">
+              <date-part name="year"/>
+            </date>
+          </else>
+        </choose>
+      </else-if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="author-year">
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" name-as-sort-order="all"/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="conditionalusetitle" font-style="italic"/>
+      </substitute>
+    </names>
+    <text macro="year" prefix=" (" suffix=")"/>
+  </macro>
+  <macro name="author-sort">
+    <names variable="author">
+      <name form="short" and="text" delimiter-precedes-last="never" et-al-min="2" et-al-use-first="1" initialize-with=". "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="conditionalusetitle"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-year-short">
+    <group delimiter=" ">
+      <text macro="author-sort"/>
+      <choose>
+        <if variable="issued original-date" match="none">
+          <text macro="year" prefix=", "/>
+        </if>
+        <else-if variable="original-date">
+          <date variable="original-date">
+            <date-part name="year"/>
+          </date>
+          <date variable="issued" prefix=" [" suffix="]">
+            <date-part name="year"/>
+          </date>
+        </else-if>
+        <else>
+          <text macro="year"/>
+        </else>
+      </choose>
+    </group>
+  </macro>
+  <macro name="title">
+    <choose>
+      <if variable="author editor translator" match="any">
+        <choose>
+          <if variable="container-title" match="none" type="thesis webpage speech musical_score legislation legal_case interview">
+            <text variable="title" font-style="italic" text-case="title"/>
+            <choose>
+              <if type="broadcast" match="any">
+                <group>
+                  <text value="DVD" prefix=", " suffix=", "/>
+                  <text value="extended" suffix=" "/>
+                  <label variable="edition" form="short"/>
```

Comment:
What exactly is this for? In general, hardcoding these words seems inappropriate (what if the citation is not to a DVD?). I would suggest instead printing the `genre` variable and expecting this information to be in the item data.